ASSETS.md

```diff
@@ -39,6 +39,26 @@ requirement:
     version.min: 7.1
   filebeat:
     version.min: 7.2
+
+# The order of the items listed here is the order they show up in the package overview.
+screenshots:
+  # The src path is relative from inside the package. The full path will be generated by the server
```

Comment:
👍 Not sure if this is a different from the `icon` behavior but I like this explicit statement. Good experience for the integration authors and registry API consumers.

Reply:
We should make the behaviour in the icon the same I think.